frontend/apps/filemanager/filemanager.lua

```diff
@@ -228,6 +240,7 @@ function FileManager:setupLayout()
     local fileManager = self
```

Comment:
(Not your code, but if you're using this, better rename to `file_manager` as fileManager is the style used to name a method/function)